### ISSUE TITLE
Unexpected ‘@’ in program. 直接在协议文件h.里面实现 报警告 求解答

### ISSUE THREAD

**Issue by ghost** [forkingdog/ProtocolKit#3]
<img width="1372" alt="00" src="https://cloud.githubusercontent.com/assets/10295701/24585382/97a0c320-17bb-11e7-8e14-c24bcba65a81.png">

@protocol Forkable <NSObject>

@optional
- (void)fork;

@required
- (NSString *)github;
- (void)jzj;
@end
// Protocol Extension
@defs(Forkable)
- (void)fork {
    NSLog(@"Forkable protocol extension: I'm forking (%@).", self.github);
}
- (void)jzj {
    
    NSLog(@"jzj");
}
- (NSString *)github {
    return @"This is a required method, concrete class must override me.";
}
@end

**Comment by beforeold** [comment]
Obviously, it is **FORBIDDEN** to implement any method in a **.h** file.
